src/collective/fingerpointing/browser/logview.pt

```diff
@@ -23,7 +23,9 @@
 
       <div id="content-core">
         <div id="layout-contents">
-          <pre tal:content="view/log_tail"></pre>
+          <pre><span tal:repeat="item view/log_tail" tal:omit-tag=""
```

Comment:
we'd end up with multiple pre tags this way @hvelarde 

i think it's perfectly fine this way although i personally prefere something like this (to avoid tal:omit)
```xml
<tal:lines repeat="item view/log_tail">
  <span tal:replace="item">2016-03-10 .....</span>
</tal:lines>
```